### ISSUE TITLE
products and sums of setoids

### ISSUE THREAD

**Issue by HuStmpHrrr** [agda/agda-stdlib#879]
it is interesting to see that there is no induced products and sums of setoids in this library yet. do I just overlook it or is it indeed the case.

**Comment by MatthewDaggitt** [comment]
If I understand what you're after correctly then they should be in 'Data.(Sum/Product).Relation.Binary.Equality.Setoid'?

**Comment by HuStmpHrrr** [comment]
hmm I don't find those modules though?

**Comment by HuStmpHrrr** [comment]
oh you meant `Pointwise`. I see, that's a bit tricky to notice.

**Comment by MatthewDaggitt** [comment]
Hmm, it definitely would be a little easier to find if the modules I suggested actually existed...

**Comment by laMudri** [comment]
[non-dependent products](https://agda.github.io/agda-stdlib/Data.Product.Relation.Binary.Pointwise.NonDependent.html#7563), [sums](https://agda.github.io/agda-stdlib/Data.Sum.Relation.Binary.Pointwise.html#6974)

But yeah, this is a classic problem of having two hierarchies (type formers and order structures), but only reflecting one of them in the module hierarchy.

**Comment by JacquesCarette** [comment]
So this issue really is about findability rather than missing functionality? I have seen sum and product of setoids in the 2.0 library (but, of course, I can't find them right now...)

**Comment by jamesmckinna** [comment]
> [non-dependent products](https://agda.github.io/agda-stdlib/Data.Product.Relation.Binary.Pointwise.NonDependent.html#7563), [sums](https://agda.github.io/agda-stdlib/Data.Sum.Relation.Binary.Pointwise.html#6974)
> 
> But yeah, this is a classic problem of having two hierarchies (type formers and order structures), but only reflecting one of them in the module hierarchy.

So can we close this in favour of a separate issue about hierarchies?


> So this issue really is about findability rather than missing functionality? 

And would the 'hierarchy' issue help/solve the findability/navigability issue?

TL;DR: suggest we close, but open new, but more appropriate issues instead?
